
[Function: __init__ start...]
Traceback (most recent call last):
  File "traineval.py", line 50, in <module>
    traindataset = GrabNetDataset(config.dataset_dir, 'train', num_mask=cfg.num_mask)
  File "/home/datassd/yilin/Codes/Hand/ConditionHOI/utils/utils.py", line 15, in function_timer
    result = function(*args, **kwargs)
  File "/home/datassd/yilin/Codes/Hand/ConditionHOI/dataset/Dataset.py", line 90, in __init__
    self.object_meshes = self.load_obj_meshes()
  File "/home/datassd/yilin/Codes/Hand/ConditionHOI/dataset/Dataset.py", line 116, in load_obj_meshes
    obj_meshes[obj_name] = trimesh.load(self.obj_mesh_dir, obj_file)
  File "/home/yilin/anaconda3/envs/pytorch3d/lib/python3.8/site-packages/trimesh/exchange/load.py", line 127, in load
    loaded = load_mesh(file_obj,
  File "/home/yilin/anaconda3/envs/pytorch3d/lib/python3.8/site-packages/trimesh/constants.py", line 153, in timed
    result = method(*args, **kwargs)
  File "/home/yilin/anaconda3/envs/pytorch3d/lib/python3.8/site-packages/trimesh/exchange/load.py", line 209, in load_mesh
    results = mesh_loaders[file_type](file_obj,
  File "/home/yilin/anaconda3/envs/pytorch3d/lib/python3.8/site-packages/trimesh/exchange/ply.py", line 100, in load_ply
    elements, is_ascii, image_name = parse_header(file_obj)
  File "/home/yilin/anaconda3/envs/pytorch3d/lib/python3.8/site-packages/trimesh/exchange/ply.py", line 382, in parse_header
    if 'ply' not in str(file_obj.readline()).lower():
AttributeError: 'str' object has no attribute 'readline'